### PR TITLE
Add footer with CC license

### PR DESCRIPTION
**Pull request by stephaniehobson**
Close mozilla/irlpodcast#124
Fix Bug #1469401

- Made footer into header
- Added site footer
- Removed privacy and cookies links from about page
- Added privacy and cookies links to footer

### CODE REVIEW

File: layouts/_default/baseof.html [hunk 1]
```diff
@@ -55,10 +55,12 @@
 </head>
 
 <body id="{{ block "bodyID" . }}home{{ end }}">
+
     <main class="page-content" aria-label="Content">
     {{ block "main" . }}{{ end }}
     </main>
 
+    {{ partial "header.html" }}
```
Comment (jpetto) [inline layouts/_default/baseof.html]: Semantically, do you think this should be above `<main>`? I'm not a huge stickler - just asking.

File: layouts/partials/footer.html [hunk 2]
```diff
@@ -1,10 +1,9 @@
 <footer id="footer">
     <div class="content">
         <ul>
-            <li><a href="/">Home</a></li>
-            <li><a href="/about">About</a></li>
-            <li><a href="https://www.mozilla.org/internet-health/" rel="external">Internet Health</a></li>
+            <li><a href="https://www.mozilla.org/privacy/">Privacy Notice</a></li>
+            <li><a href="https://www.mozilla.org/privacy/websites/#cookies">Cookies</a></li>
         </ul>
+        <p>Unless otherwise indicated, IRL Podcast episodes are available under the <a href="https://creativecommons.org/licenses/by-sa/3.0/us/">Creative Commons Attribution Share-Alike 3.0</a> Unported license, or any later version. </p>
```
Comment (jpetto) [inline layouts/partials/footer.html]: Add `rel="external"` to this link?

File: src/scss/components/_header.scss [hunk 3]
```diff
@@ -0,0 +1,45 @@
...
+        color: #fff;
+        text-decoration: none;
+    }
+
+    a:focus,
+    a:hover,
+    a:visited:focus,
+    a:visited:hover {
+        text-decoration: underline;
+    }
```
Comment (jpetto) [inline src/scss/components/_header.scss]: Maybe abstract these link styles for more DRY-ness?

File: layouts/_default/baseof.html (hunk 1 above)
Comment (stephaniehobson) [reply layouts/_default/baseof.html]: I assumed it was last for REASONS and left it down there. (Maybe to do with it being sticky?) I can move it up and test though.

**Comment by stephaniehobson** [comment]
Updated.

**Pull request merged**